### ISSUE TITLE
Finetuning Parameters

### ISSUE THREAD

**Issue by corypaik** [allenai/unifiedqa#16]
Hi, I was hoping to get some input on the parameters used for finetuning UnifiedQA to reproduce the experiments in Section 6.3 of the paper. I'm using the [t5-trivia notebook provided in the T5 repository](https://github.com/google-research/text-to-text-transfer-transformer/blob/732efb09be3c6ed9b2d0d1b3e6ac45f08e15989a/notebooks/t5-trivia.ipynb), with the task setup functions and preprocessed data that you've provided. Do these seem like the correct parameters for doing so? 

```python
# Set parallelism and batch size to fit on v2-8 TPU (if possible).
model_parallelism, train_batch_size = {
    "small": (1, 256),
    "base": (2, 128),
    "large": (8, 64),
    "3B": (8, 16),
    "11B": (8, 16)}[MODEL_SIZE]

model = t5.models.MtfModel(
    model_dir=MODEL_DIR,
    tpu=TPU_ADDRESS,
    tpu_topology=TPU_TOPOLOGY,
    model_parallelism=model_parallelism,
    batch_size=train_batch_size,
    sequence_length={"inputs": 512, "targets": 100},
    learning_rate_schedule=0.003,
    save_checkpoints_steps=2000,
    keep_checkpoint_max=None,
    iterations_per_loop=100,
)
...
model.finetune(
    mixture_or_task_name="physical_iqa_mixture",
    pretrained_model_dir=PRETRAINED_DIR,
    finetune_steps=60000
)
```

Thank you for your help!

**Comment by danyaljj** [comment]
Looks reasonable! 

I used the following hyperparams: 
```
HYPERPARAMETER_DICT = {
    "small": (1, 256, 40),
    "base": (2, 128, 28),
    "large": (8, 64, 24),
    "3B": (8, 16, 30),
    "11B": (8, 8, 30)
}
```

And 100k fine-tuning steps. 
And *I think* we used a learning rate of `learning_rate_schedule=0.001`.
